<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqttDemoClass</class>
 <widget class="QMainWindow" name="MqttDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MqttDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Receive</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IpEdit">
      <property name="text">
       <string>192.168.31.101</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="5">
     <widget class="QLineEdit" name="SendMessageEdit"/>
    </item>
    <item row="3" column="1" colspan="6">
     <widget class="QPlainTextEdit" name="ReceiveMessageEdit"/>
    </item>
    <item row="0" column="5" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="PortEdit">
      <property name="text">
       <string>1883</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="SendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="ConnectControlButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Topic</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="SubscribeButton">
      <property name="text">
       <string>Subscribe</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="5">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MqttDemo.qrc"/>
 </resources>
 <connections/>
</ui>
